run.sh

```diff
@@ -132,7 +124,7 @@ main() {
         done
     fi
 
-    h1 'WordPress Configuration Complete!'
+    h1 'WordPress Installation Complete!'
```

Comment:
I've had two people I work with comment about the language here. So I switched it to match the initial `h1` language.